StkAutomation/Matlab/STK Object Model/Radar_Scheduler_Deck_Access.m

```diff
@@ -0,0 +1,163 @@
+% %Written By: Alexander Ridgeway
+% %Date:       27 April 2016
+% 
+clear;
+clc;
+try
+    % Grab an existing instance of STK
+    uiapp = actxGetRunningServer('STK12.application');
+catch
+    % STK is not running, launch new instance
+    % Launch a new instance of STK and grab it
+    uiapp = actxserver('STK12.application');
+end
+
+%get the root from the personality
+root = uiapp.Personality2;
+% set visible to true (show STK GUI)
+uiapp.visible = 1;
+%%From the STK Object Root you can command every aspect of the STK GUI
+%close current scenario or open new one
+try
+    root.CloseScenario();
+    root.NewScenario('DeckAccess');
+catch
+    root.NewScenario('DeckAccess');
+end
+%get the scenario root, its of type IAgScenario 
+scen = root.CurrentScenario;
+%root.ExecuteCommand('SetUnits / Km Epsec');
+
+%Create the Radar Site (Delete if it exists)
+try
+    fac = root.CurrentScenario.Children.New('eFacility','Radar');
+catch
+    fac = root.GetObjectFromPath('/Facility/Radar');
+    fac.Unload;
+    fac = root.CurrentScenario.Children.New('eFacility','Radar');
+end
+
+FacLat = 13.7355;
+FacLon = 80.1847;
+FacAlt = -0.0564551;
+RangeMax = 1000;
+Start = '1 Jan 2017 00:00:00.000';
+Stop = '1 Jan 2017 00:30:00.000';
+
+
+fac.UseTerrain = false;
+fac.Position.AssignGeodetic(FacLat,FacLon,FacAlt);
+sen = fac.Children.New('eSensor','Dome');
+sen.CommonTasks.SetPatternRectangular(65,65);
+const = sen.AccessConstraints;
+range = const.AddConstraint(34); 
```

Comment:
Use the accessConstraint enumeration member rather than the enumeration value 

`range = const.AddConstraint('eCstrRange')`